### PR TITLE
Its possible to change composer filename

### PR DESCRIPTION
**Pull request by sergeyz**
https://getcomposer.org/doc/03-cli.md#composer

### CODE REVIEW

**Comment by HeahDude** [comment]
👍

File: Resources/skeleton/app/SymfonyRequirements.php [hunk 1]
```diff
@@ -783,7 +783,12 @@ protected function getRealpathCacheSize()
      */
     protected function getPhpRequiredVersion()
     {
-        if (!file_exists($path = __DIR__.'/../composer.lock')) {
+        $lockFile = 'composer.lock';
+        if ($jsonFile = getenv('COMPOSER')) {
+            $lockFile = str_replace('.json', '.lock', $jsonFile);
```
Comment (xabbuh) [inline Resources/skeleton/app/SymfonyRequirements.php]: I would only replace this string when it is the filename extension.

**Comment by HeahDude** [comment]
ping @sergeyz :)

**Comment by fabpot** [comment]
Closing as I don't think we should support this possibility.

**Pull request closed without merging**